### PR TITLE
AC Test Fix - LRAC-9383 | master 

### PR DESCRIPTION
**Pull request by manuferreira**
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1610789301&testrayRunId=1610790232 (Testray run)
https://issues.liferay.com/browse/LRAC-9383 (Bug ticket)

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase [hunk 1]
```diff
@@ -59,15 +59,10 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8626 | Test Summary: Segment distribution chart can be filtered by date property"
-	@ignore = "true"
 	@priority = "5"
 	test CanDistributionChartFilteredDateProperty {
 		property portal.upstream = "quarantine";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:64]: Hey @manuferreira, I think you got confused here, this [test passed when you ran on CI](https://testray.liferay.com/home/-/testray/case_results/1610790662?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D1610789301%26testrayRunId%3D1610790232%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:64]: Yes, lol. Thanks for pointing that out :smile_cat:

**Comment by manuferreira** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]

**Review by alexcandidos — APPROVED** [review]

**Comment by alexcandidos** [comment]
Hi @manuferreira , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**